hmda/src/main/scala/hmda/validation/rules/lar/validity/common/V631_1.scala

```diff
@@ -25,7 +25,10 @@ object V631_1 extends EditCheck[LoanApplicationRegister] {
   )
 
   override def apply(lar: LoanApplicationRegister): ValidationResult =
-    when(lar.coApplicant.ethnicity.otherHispanicOrLatino is empty) {
-      lar.coApplicant.ethnicity.ethnicity1 is containedIn(validEthnicities)
+    if(lar.coApplicant.ethnicity.otherHispanicOrLatino.isEmpty) {
+      lar.coApplicant.ethnicity.ethnicity1 is containedIn(validEthnicities) and (lar.coApplicant.ethnicity.ethnicity1 is equalTo(EmptyEthnicityValue))
```

Comment:
the and should be and not though you can remove the whole `and (lar.applicant.ethnicity.ethnicity1 is equalTo(EmptyEthnicityValue)` clause since it's redundant anyway